config/md5

```diff
@@ -1202,6 +1202,18 @@ https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/20.04/x86_64/ruby-3.0.0.tar
 https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.04/x86_64/ruby-3.1.3.tar.bz2=2be294a056a45a50262d1796906f8860
 https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.04/x86_64/ruby-3.2.0.tar.bz2=33db4a687e4943faf6c2853179b67935
 https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.04/x86_64/ruby-3.2.1.tar.bz2=dea6663b2c113190e9b99a6dcedb3aa0
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.04/aarch64/ruby-2.7.6.tar.bz2=07b12bd2a7d8242c34d824882e654c22
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.04/aarch64/ruby-2.7.7.tar.bz2=fe12c0e211f90076b51c7916c830971f
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.04/aarch64/ruby-3.0.4.tar.bz2=65ab3105b0c6d5e8a2e9977a3ab7ed94
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.04/aarch64/ruby-3.0.5.tar.bz2=08547ad8e760e0776c74401ae5bbd8e5
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.04/aarch64/ruby-3.1.2.tar.bz2=631b4ea065d18df96c6ccbc141904bb3
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.04/aarch64/ruby-3.1.3.tar.bz2=03a67e234d886f3c698c1571e42cc7a6
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.04/aarch64/ruby-3.2.0.tar.bz2=0ded0cb8ce18a846d71d7e04d24c2e78
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.04/aarch64/ruby-3.2.1.tar.bz2=5251aed0ba4d8349413d6bf4c261bea3
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.10/x86_64/ruby-3.0.5.tar.bz2=f09694c4f23d7625f72606bce0efb710
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.10/x86_64/ruby-3.1.3.tar.bz2=340db3acb58829b51e4a7ac1c77246d4
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.10/x86_64/ruby-3.2.0.tar.bz2=a1bd86cb46a9e6c40c285258879a3d59
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.10/x86_64/ruby-3.2.1.tar.bz2=d86b2a1e9721b9459e1283006e03fa92
```

Comment:
`3.1.3`, `3.2.0` and `3.2.1` are duplicate (see lines `1202`, `1203` and `1204.

Reply:
No, these are for Ubuntu 22.10, Kinetic